scripts/install_deps.sh

```diff
@@ -133,21 +133,22 @@ case $(uname -s) in
 # Arch Linux
 #------------------------------------------------------------------------------
 
-            Arch)
+            Arch*)
```

Comment:
Can you leave a comment what is the name on your system (if its not just `Arch`)?

Reply:
It's probably `Arch Linux` (at least on mine it is)

Reply:
yeah it's Arch linux:

```
> more /etc/os-release
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
ID_LIKE=archlinux
ANSI_COLOR="0;36"
HOME_URL="https://www.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://bugs.archlinux.org/"
```